### PR TITLE
Update throttling limits for display package page

### PR DESCRIPTION
**Pull request by shishirx34**

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[e3e38a7](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e3e38a7b1a41b89d10bf1a627dce5353606a79bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005062106476627-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=I1qp1diP8GR3TuvCGsAp8IdUHQ5fuzIcceOd%2B3AVu5w%3D&st=2020-05-06T21%3A02%3A35Z&se=2020-06-06T21%3A07%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/rate-limits.md [hunk 1]
```diff
@@ -51,3 +51,9 @@ The following tables list the rate limits for the NuGet.org API.
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+| API | Limit Type | Limit Value | API usecase | 
+|:---|:---|:---|:--- |
+**GET** `/package/{id}/{version}` | IP | 50 / minute | Display package (version) details page.
```
Comment (joelverhagen) [inline docs/api/rate-limits.md:59]: Maybe say something like:

`
If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 API](relative-path-here). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interactio.
`

.. or something like that.

File: docs/api/rate-limits.md [hunk 2]
```diff
@@ -51,3 +51,9 @@ The following tables list the rate limits for the NuGet.org API.
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+| API | Limit Type | Limit Value | API usecase |
```
Comment (joelverhagen) [inline docs/api/rate-limits.md:57]: I think `use case` is two words

Comment (shishirx34) [reply docs/api/rate-limits.md:57]: whoops.. right, needs fixing in the rest of the document too.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[1eddda1](https://github.com/NuGet/docs.microsoft.com-nuget/commits/1eddda1cd068a9ae8274bc55618a7c77c34c1807)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005062237559200-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=I0%2Bc3iIcpZd3fogER7vJbTcVjh15g76nxsLdbLdo4Yk%3D&st=2020-05-06T22%3A33%3A41Z&se=2020-06-06T22%3A38%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/rate-limits.md [hunk 3]
```diff
@@ -47,7 +47,15 @@ The following tables list the rate limits for the NuGet.org API.
...
 
-| API | Limit Type | Limit Value | API usecase | 
+| API | Limit Type | Limit Value | API use case | 
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 search APIs](search-query-service-resource.md). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interaction.
```
Comment (joelverhagen) [inline docs/api/rate-limits.md:57]: Let's link to the overview not search specifically:
https://docs.microsoft.com/en-us/nuget/api/overview

**Comment by anangaur** [comment]
Docs Build status updates of commit _[e4365d4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e4365d4be0f4a31013e0c96efb927e828046c879)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) | [Details](#user-content-38ce0270-b429e171)

<a id="38ce0270-b429e171"></a>
### [docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md)
 - **Line 57**: **[Warning]** `````Link 'https://docs.microsoft.com/en-us/nuget/api/overview' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005062252065968-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cqDPY91PXuz3Wdg6rBmmhjsr%2FHYvAvdWouBs8V90kdc%3D&st=2020-05-06T22%3A47%3A52Z&se=2020-06-06T22%3A52%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[7bb5542](https://github.com/NuGet/docs.microsoft.com-nuget/commits/7bb5542d3fb6cc2fb6d21f544e20b5927cb1f409)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C6%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005062254538175-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FaFfds00iwIdXnK2cdza9p3sSM6U48fGLUw%2FGcipXu4%3D&st=2020-05-06T22%3A50%3A41Z&se=2020-06-06T22%3A55%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/rate-limits.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ The following tables list the rate limits for the NuGet.org API.
 
 ## NuGet Gallery page views
 
-If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 search APIs](search-query-service-resource.md). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interaction.
+If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 APIs](https://docs.microsoft.com/en-us/nuget/api/overview). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interaction.
```
Comment (joelverhagen) [inline docs/api/rate-limits.md]: This should be a relative link, similar to what you have before.

File: docs/api/rate-limits.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ The following tables list the rate limits for the NuGet.org API.
 > [!Note]
 > We recommend using NuGet.org's [V3 search APIs](search-query-service-resource.md) as it is not rate limited currently. For V1 and V2 search APIs, the following limits apply:
 
-| API | Limit Type | Limit Value | API usecase |
+| API | Limit Type | Limit Value | API use case |
```
Comment (loic-sharma) [inline docs/api/rate-limits.md:41]: Should we have consistent casing for the headings? Either have `Limit type` and `Limit value`, or, change your heading to `API Use Case`. Same comment for line 50

File: docs/api/rate-limits.md (hunk 1 above)
Comment (loic-sharma) [reply docs/api/rate-limits.md:61]: Are people aware that "NuGet Gallery" means the nuget.org website? Can we just say nuget.org website instead?

**Comment by anangaur** [comment]
Docs Build status updates of commit _[a4d96c4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/a4d96c47679ba90694ce506a0d3ccdc21d8bbe40)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005072025424884-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ozUdWebb0F7tQZeA4tqP%2BzuhsEdczcQUp4EHv4FLrrA%3D&st=2020-05-07T20%3A21%3A32Z&se=2020-06-07T20%3A26%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/rate-limits.md [hunk 6]
```diff
@@ -47,7 +47,15 @@ The following tables list the rate limits for the NuGet.org API.
...
 
-| API | Limit Type | Limit Value | API usecase | 
+| API | Limit Type | Limit Value | API Use Case | 
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 APIs](overview.md). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interaction.
```
Comment (loic-sharma) [inline docs/api/rate-limits.md:57]: It looks like this comment may have been missed:

> Are people aware that "NuGet Gallery" means the nuget.org website? Can we just say nuget.org website instead?

Comment (shishirx34) [reply docs/api/rate-limits.md:57]: I did miss it. will change it.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[e270e64](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e270e64a87164f9b31f931dd12e3a2f73146eb05)_: 

:clock10: **Pending**: waiting for processors ([39](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 39 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by anangaur** [comment]
Docs Build status updates of commit _[e270e64](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e270e64a87164f9b31f931dd12e3a2f73146eb05)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005112000308139-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2bSBgh%2BSUd%2FenH4VcW5fGrKJNJnBnCRIWk1a4VjgUzQ%3D&st=2020-05-11T19%3A56%3A24Z&se=2020-06-11T20%3A01%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by anangaur** [comment]
Docs Build status updates of commit _[321f5cc](https://github.com/NuGet/docs.microsoft.com-nuget/commits/321f5cc998a41062bde603f74246d290c6c19d50)_: 

:clock10: **Pending**: waiting for processors ([39](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 39 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by anangaur** [comment]
Docs Build status updates of commit _[321f5cc](https://github.com/NuGet/docs.microsoft.com-nuget/commits/321f5cc998a41062bde603f74246d290c6c19d50)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/rate-limits.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dp-rate-limit/docs/api/rate-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/rate-limits?branch=pr-en-us-2028) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C11%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202005112001437735-2028%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=r8siE7shaBtWOJwi12NW8%2BDqHaMNPf%2Bj5LMFDQxpMYo%3D&st=2020-05-11T19%3A57%3A32Z&se=2020-06-11T20%3A02%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**